tests/PHPStan/Analyser/data/literal-string.php

```diff
@@ -27,6 +27,9 @@ public function doFoo($literalString, string $string)
 		assertType('literal-string', str_repeat($literalString, 10));
 		assertType("''", str_repeat('', 10));
 		assertType("'foofoofoofoofoofoofoofoofoofoo'", str_repeat('foo', 10));
+		$str = str_repeat('a', 99);
+		assertType("'$str'", $str);
```

Comment:
Alright, one more thing, sorry. This isn't a useful test, it's a tautology and we don't know what's getting tested here. Just repeat 'a' 99 times in the string, thanks.